database/seeders/GenreSeeder.php

```diff
@@ -13,6 +15,25 @@ class GenreSeeder extends Seeder
      */
     public function run()
     {
-        //
+        $categories = [
+            "Action",
+            "Anime",
+            "Adventure",
+            "Comedy",
+            "Classic",
+            "Crime",
+            "Drama",
+            "Fantasy",
+            "Horror",
+            "Mystery",
+            "Romance",
+            "Thriller"
+        ];
+
+        foreach($categories as $category) {
+            DB::table('genres')->insert([
+                'name' => $category
+             ]);
```

Comment:
Here if you have the defined Factory for Genre, you can call the factory here:

```
foreach($genres as $genre) {
                 Genre::factory()->create([ 
                     'name' => $genre
                  ])
            }
```
Make sure you import the Genre factory in this file 😄 

I think that in the future the categories could be a constant in the Genre model too. This makes the code a bit more maintainable. 

So on your Genre model you could have:

```
const GENRE_ACTION = 'Action';
.....
```

Then in the genre array, you could change the names for `Genre::GENRE_ACTION`

By doing this you would not need the DB class anymore, so that can be removed from the import section of this file.